src/ssh/sftp_client.cpp

```diff
@@ -29,23 +27,21 @@
 #include <fcntl.h>
 
 #include <QFile>
-#include <QTextStream>
 
 namespace mp = multipass;
 
 namespace
 {
 constexpr int file_mode = 0664;
```

Comment:
Shouldn't we keep the same permissions as the original, at least if they were stricter?

Reply:
Sure, for 'push_file' and 'pull_file', but for streaming, we'll have to make a best guess like this.

I'll fix the `push_file` & `pull_file` cases.

Reply:
Actually, let's defer this until we use `push_file` and `pull_file` here as the defacto way of transferring files.  The problem is, Qt uses a different file permissions enum than what is common in Posix, so we have to translate between the two.  We have this code already in `src/sshfs_mount/sftp_server.cpp`, but we'll need to commonize that code for these purposes and I'd like to leave that for another PR.

Reply:
OK, fine by me.